### ISSUE TITLE
panic: runtime error on: task --list

### ISSUE THREAD

**Issue by fjmk** [go-task/task#99]
- Task version
task --version
Task version: SNAPSHOT-d48a2f3ccfe779b7631798d0d2e18e6e94ea4296 
from [task_linux_amd64.deb version v2.0.0](https://github.com/go-task/task/releases/download/v2.0.0/task_linux_amd64.deb)

- OS
Ubuntu 16.04.4 LTS

- Steps to reproduce the issue:
```bash
mkdir test
cd test
task --init
task --list
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x55d20b]

goroutine 1 [running]:
github.com/go-task/task/internal/logger.(*Logger).Outf(0x0, 0x760b40, 0x29, 0x0, 0x1, 0x1)
        /home/frans/go/src/github.com/go-task/task/internal/logger/logger.go:18 +0x8b
github.com/go-task/task.(*Executor).PrintTasksHelp(0xc42013e000)
        /home/frans/go/src/github.com/go-task/task/help.go:15 +0x234
main.main()
        /home/frans/go/src/github.com/go-task/task/cmd/task/task.go:106 +0x9bb
```

- Generated Taskfile.yml:
```yaml
version: '2'

vars:
  GREETING: Hello, World!

tasks:
  default:
    cmds:
      - echo "{{.GREETING}}"
    silent: true
```

**Comment by harobed** [comment]
Same on OSX:

```
$ task --version
Task version: SNAPSHOT-d48a2f3ccfe779b7631798d0d2e18e6e94ea4296
```

**Comment by andreynering** [comment]
Thanks for reporting. Fixed on v2.0.1
